AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -845,6 +845,7 @@ mihoyo.com##.mihoyo-cookie-tips
 toplitz-productions.com##.mod_cookiebar_optin
 about.me##div[class^="cookie_notice-scope-"]
 dell.com,xn--allestrungen-9ib.de,reuters.com###_evh-button
+pfizeryaninizda.com###_evh-link
```

Comment:
We have a common rule `##.evidon-consent-button`. Exchange your rule by adding a domain to this one.